docs/index.rst

```diff
@@ -1,18 +1,13 @@
-.. include:: ../README.rst
-
-API documentation
-=================
-
-.. automodule:: hca.dss
-   :members:
-   :inherited-members:
+=======
+HCA CLI
+=======
 
 Table of Contents
 =================
 
 .. toctree::
    :maxdepth: 5
 
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
+   readme
+   cli
+   api
```

Comment:
This makes the index page only contain a table of contents. Please change this so that the introduction still appears on the index page.

Reply:
added readme.rst to index.rst